kfs-core/src/main/resources/edu/arizona/kfs/sys/document/validation/configuration/FinancialSystemValidators.xml

```diff
@@ -0,0 +1,81 @@
+    <property name="fundGroupAllowedValidation" ref="AccountingDocument-IsFundGroupAllowed-DefaultValidation"/>
+    <property name="subFundGroupAllowedValidation" ref="AccountingDocument-IsSubFundGroupAllowed-DefaultValidation"/>
+    <property name="objectSubTypeAllowedValidation" ref="AccountingDocument-IsObjectSubTypeAllowed-DefaultValidation"/>
+    <property name="objectLevelAllowedValidation" ref="AccountingDocument-IsObjectLevelAllowed-DefaultValidation"/>
+    <property name="objectConsolidationAllowedValidation" ref="AccountingDocument-IsObjectConsolidationAllowed-DefaultValidation"/>
+    <!-- KITT-1934 -->
+    <property name="globalTransactionValidation" ref="AccountingDocument-GlobalTransactionValidation"/>
+  </bean>
+
+  <!-- KITT-803 -->
+  <!-- will only do sub type sub fund validation -->
+  <bean id="AccountingDocument-globalTransactionAccountingLineValuesAllowedValidation" class="edu.arizona.kfs.sys.document.validation.impl.AccountingLineValuesAllowedValidationHutch" abstract="true">
+    <property name="globalTransactionValidation" ref="AccountingDocument-GlobalTransactionValidation"/>
+  </bean>
+
+  <!-- KITT-296, KITT-803 -->
+  <bean id="AccountingDocument-GlobalTransactionValidation" scope="prototype" parent="AccountingDocument-GlobalTransactionValidation-parentBean"/>
+
+  <bean id="AccountingDocument-GlobalTransactionValidation-parentBean" class="edu.arizona.kfs.sys.document.validation.impl.AccountingLineGlobalTransactionValidation" abstract="true">
+    <property name="parameterService" ref="parameterService"/>
+    <property name="globalTransactionEditService" ref="globalTransactionEditService"/>
+    <property name="docTypesToValidate">
+      <list>
+        <value>AdvanceDepositDocument</value>
+        <value>AssetPaymentDocument</value>
+        <value>AuxiliaryVoucherDocument</value>
+        <value>BudgetAdjustmentDocument</value>
+        <value>BenefitExpenseTransferDocument</value>
+        <value>CashReceiptDocument</value>
+        <value>CreditCardReceiptDocument</value>
+        <value>CustomerInvoiceDocument</value>
+        <value>DisbursementVoucherDocument</value>
+        <value>DistributionOfIncomeAndExpenseDocument</value>
+        <value>GeneralErrorCorrectionDocument</value>
+        <value>IndirectCostAdjustment</value>
+        <value>InternalBillingDocument</value>
+        <value>JournalVoucherDocument</value>
+        <value>NonCheckDisbursement</value>
+        <value>LaborJournalVoucherDocument</value>
+        <value>PaymentRequestDocument</value>
+        <value>PreEncumbranceDocument</value>
+        <value>ProcurementCardDocument</value>
+        <value>PurchaseOrderDocument</value>
+        <value>RequisitionDocument</value>
+        <value>SalaryExpenseTransferDocument</value>
+        <value>ServiceBillingDocument</value>
+        <value>UaRequisitionDocument</value>
+        <value>UaDisbursementVoucherDocument</value>

```

Comment:
UaRequisitionDocument and UaDisbursementVoucherDocument do not exist in our KFS 6.0 code. These documents can be removed from the list.